src/views/pearTemplate.hbs

```diff
@@ -0,0 +1,3 @@
+<main class = 'pear-container'>
+    {{> pear this}}
+</main>
```

Comment:
same as above

Reply:
this was in the tweeter thing so i went ahead and added it, ill remove this file for now